qdec in same70q21b not compiling, however in same70q21 works

It looks like there is an issue with using the QDEC in **same70q21b** vs **same70q21**.

When compiling with enabled `sam-qdec` I get an issue in that file:
[drivers/sensor/qdec_sam/qdec_sam.c](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/sensor/qdec_sam/qdec_sam.c)

missing definition of `TC_CMR_ETRGEDG_NONE`

```
.../zephyr/drivers/sensor/qdec_sam/qdec_sam.c: In function 'qdec_sam_configure':
.../zephyr/drivers/sensor/qdec_sam/qdec_sam.c:90:27: error: 'TC_CMR_ETRGEDG_NONE' undeclared (first use in this function); did you mean 'TC_CMR_ETRGEDG_Pos'?
   90 |                         | TC_CMR_ETRGEDG_NONE

```

this definition is present in:

[asf/sam/include/same70/component/tc.h](https://github.com/zephyrproject-rtos/hal_atmel/blob/master/asf/sam/include/same70/same70q21.h)

but **not** present in:
[asf/sam/include/same70b/component/tc.h](https://github.com/zephyrproject-rtos/hal_atmel/blob/master/asf/sam/include/same70b/same70q21b.h)


steps to reproduce
cmake
`-DBOARD=sam_e70b_xplained` (**fails**)  vs `-DBOARD=sam_e70_xplained` (**works**)

dt
```
&tc0 {
	status = "okay";
	compatible = "atmel,sam-tc-qdec";

	pinctrl-0 = <&tc0_qdec_default>;
	pinctrl-names = "default";
	label = "QDEC_0";
};
```
conf
```
# quadrature decoder
CONFIG_SENSOR=y
CONFIG_QDEC_SAM=y

```
from the datasheet, I dont see any difference between both revision regarding qdec:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/891378/81666bd3-053a-4765-985b-e4524317f225)

I suggest to copy paste from 
`same70/same70q21.h`
to
`same70b/same70q21b.h`

--- a/boards/arm/sam_e70_xplained/sam_e70_xplained-pinctrl.dtsi
+++ b/boards/arm/sam_e70_xplained/sam_e70_xplained-pinctrl.dtsi
@@ -83,6 +83,27 @@
 		};
 	};
 
+	tc1_qdec_default: tc1_qdec_default {
+		group1 {
+			pinmux = <PC23B_TC1_TIOA3>,
+				 <PC24B_TC1_TIOB3>;
+		};
+	};
+
+	tc2_qdec_default: tc2_qdec_default {
+		group1 {
+			pinmux = <PC5B_TC2_TIOA6>,
+				 <PC6B_TC2_TIOB6>;
+		};
+	};
+
+	tc3_qdec_default: tc3_qdec_default {
+		group1 {
+			pinmux = <PE0B_TC3_TIOA9>,
+				 <PE1B_TC3_TIOB9>;
+		};
+	};
+
 	twihs0_default: twihs0_default {
 		group1 {
 			pinmux = <PA4A_TWI0_TWCK>,

--- a/boards/arm/sam_e70_xplained/sam_e70_xplained.dts
+++ b/boards/arm/sam_e70_xplained/sam_e70_xplained.dts
@@ -23,3 +23,27 @@
 	pinctrl-0 = <&tc0_qdec_default>;
 	pinctrl-names = "default";
 };
+
+&tc1 {
+	status = "disabled";
+	compatible = "atmel,sam-tc-qdec";
+
+	pinctrl-0 = <&tc1_qdec_default>;
+	pinctrl-names = "default";
+};
+
+&tc2 {
+	status = "disabled";
+	compatible = "atmel,sam-tc-qdec";
+
+	pinctrl-0 = <&tc2_qdec_default>;
+	pinctrl-names = "default";
+};
+
+&tc3 {
+	status = "disabled";
+	compatible = "atmel,sam-tc-qdec";
+
+	pinctrl-0 = <&tc3_qdec_default>;
+	pinctrl-names = "default";
+};

--- a/west.yml
+++ b/west.yml
@@ -147,7 +147,7 @@ manifest:
       groups:
         - hal
     - name: hal_atmel
-      revision: b0cc87f2a494d8b3c383292d1d6c7d10323932bd
+      revision: aad79bf530b69b72712d18873df4120ad052d921
       path: modules/hal/atmel
       groups:
         - hal

